### PR TITLE
Updates documentation to have a full working example.

### PR DESCRIPTION
**Pull request by xavier-contreras**
* Updates all docs and code references to local-routing to  network-instance instead
* Updates generate_bindings.sh to clone the models repo entirely since  models will keep changing making this a slightly more future proof way  to reference models and generate bindings

### CODE REVIEW

**Comment by xavier-contreras** [comment]
Fixes https://github.com/robshakir/pyangbind/issues/334

File: docs/example/oc-network-instance/generate_bindings.sh
Comment (JoseIgnacioTamayo) [inline docs/example/oc-network-instance/generate_bindings.sh:1]: This could be achieved with ' $(dirname $0)'

Comment (JoseIgnacioTamayo) [inline docs/example/oc-network-instance/generate_bindings.sh:1]: To make full use of $SDIR:
´´´
pyang --plugindir $PYBINDPLUGIN -f pybind -o $SDIR/binding.py  -p $DDIR/ $DDIR/network-instance/openconfig-network-instance.yang
´´´

Comment (JoseIgnacioTamayo) [inline docs/example/oc-network-instance/generate_bindings.sh:1]: This could be achieved with ' $(dirname $0)'

Comment (JoseIgnacioTamayo) [inline docs/example/oc-network-instance/generate_bindings.sh:1]: To make full use of $SDIR:
´´´
pyang --plugindir $PYBINDPLUGIN -f pybind -o $SDIR/binding.py -p $DDIR/ $DDIR/network-instance/openconfig-network-instance.yang
´´´

Comment (JoseIgnacioTamayo) [inline docs/example/oc-network-instance/generate_bindings.sh:1]: It would be useful to delete $DDIR at the end, once binding.py is created.

**Review by JoseIgnacioTamayo — COMMENTED** [review]
I suggest a few small improvements to the shell script.

**Review by JoseIgnacioTamayo — APPROVED** [review]
LGTM

File: docs/example/oc-network-instance/generate_bindings.sh
Comment (xavier-contreras) [reply docs/example/oc-network-instance/generate_bindings.sh:1]: Implemented all these changes, thanks!

**Pull request merged**